angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js

```diff
@@ -22,8 +22,172 @@ module.exports = function () {
         return tabs;
     };
 
+    var setVisibility = function (defaultParams, params, config) {
+
+        if (!_.isEqual(defaultParams, params)) {
+            config.visibility = true;
+        }
+
+        return config;
+    }
+
+    var setResultsCount = function (config, data) {
+
+        var from = ((data.page * data.perPage) - data.perPage + 1) || configService.getData(
```

Comment:
You should ask yourself "why my lines so long?" instead off writing them in few lines